README.md

```diff
@@ -11,8 +11,7 @@ the [Conduit](https://github.com/conduitio/conduit) connector protocol in gRPC.
 It also contains a thin Go layer that hides the gRPC implementation details
 without adding any functionality on top.
 
-This repository is the only connection point between Conduit and a connector
-connector.
+This repository is the only connection point between Conduit and a connector.
 
 ## Implementing a connector in Go
```

Comment:
There's an outdated link in line 30, can you please change it to "https://docs.buf.build/generate/usage#31-create-a-bufgenyaml" 🙏 

Also we should change the example `buf.gen.yaml` to use remote plugins, otherwise the user needs to have `protoc` installed as well:
```yaml
version: v1
plugins:
  # C++
  - plugin: buf.build/grpc/cpp
    out: gen/proto/cpp
  - plugin: buf.build/protocolbuffers/cpp
    out: gen/proto/cpp
  # Java
  - plugin: buf.build/grpc/java
    out: gen/proto/java
  - plugin: buf.build/protocolbuffers/java
    out: gen/proto/java
```